pkg/arvo/sys/vane/gall.hoon

```diff
@@ -1130,23 +1189,22 @@
         ==
       ::
           %pass
-        =/  =duct  system-duct.state
-        =/  =wire  p.card
-        =/  =neet  q.card
+        =/  [=wire =neet]  [p q]:card
```

Comment:
Since p and q are already in the subject, is there a reason not to use `=*` here? We're no longer modifying `wire` in place

Reply:
well, there is a `=.` a couple lines later, but yeah, using a `=*` is probably better here